api/src/main/java/org/openmrs/api/impl/GlobalLocaleList.java

```diff
@@ -33,16 +33,10 @@
 	public void globalPropertyChanged(GlobalProperty newValue) {
 		allowedLocales = new LinkedHashSet<>();
 		for (String allowedLocaleString : newValue.getPropertyValue().split(",")) {
-			try {
-				Locale allowedLocale = LocaleUtility.fromSpecification(allowedLocaleString.trim());
-				if (allowedLocale != null) {
-					allowedLocales.add(allowedLocale);
-				}
-			}
-			catch (Exception e) {
-				// bad locale spec? just ignore it. the UI should take care of
-				// guiding the user.
-			}
+			Locale allowedLocale = LocaleUtility.fromSpecification(allowedLocaleString.trim());
+        		if (allowedLocale != null) {
+				allowedLocales.add(allowedLocale);
+                        }
```

Comment:
@PKatGITHUB can you plese fix the indentation.

https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE